js2-mode.el

```diff
@@ -6163,8 +6163,6 @@ its relevant fields and puts it into `js2-ti-tokens'."
                              (setf (js2-token-beg token) (- js2-ts-cursor 2))
                              (js2-skip-line)
                              (setf (js2-token-comment-type token) 'line)
-                             ;; include newline so highlighting goes to end of window
```

Comment:
Including the newline is somewhat important actually (syntax-based font-locking does that).

I think this piece of code should look if there's a newline after, and if so, include it.